content/workshops/jboss-migration-toolkit/lab1-welcome.md

```diff
@@ -0,0 +1,99 @@
+
+##  Ability to analyze source code or application archives
+RHAMT can evaluate application archives or source code, and can evaluate multiple applications together. It can identify archives that are shared across multiple applications, which can help with more accurate effort estimation. 
+
+# RHMAT Rules
+ Red Hat Application Migration Toolkit (RHAMT) contains rule-based migration tools that analyze the APIs, technologies, and architectures used by the applications you plan to migrate. In fact, the RHAMT analysis process is implemented using RHAMT rules. RHAMT uses rules internally to extract files from archives, decompile files, scan and classify file types, analyze XML and other file content, analyze the application code, and build the reports.
+
+RHAMT builds a data model based on the rule execution results and stores component data and relationships in a graph database, which can then be queried and updated as needed by the migration rules and for reporting purposes. 
+
+RHAMT rules use the following rule pattern: 
+
+~~~~
+  when(condition)
+      perform(action)
+  otherwise(action)
+~~~~
+
+RHAMT provides a comprehensive set of standard migration rules out-of-the-box. Because applications may contain custom libraries or components, RHAMT allows you to write your own rules to identify use of components or software that may not be covered by the existing ruleset. If you plan to write your own custom rules, see the [Rules Development Guide][3] for detailed instructions. 
+
+# Who is this worksop designed to inform?
+This workshop is for engineers, consultants, and others who want to use Red Hat Application Migration Toolkit (RHAMT) to migrate Java applications or other components. It provides an overview of the Red Hat Application Migration Toolkit and how to get started using the tools to plan and execute your migration. 
+
+# Story Points
+Story points are an abstract metric commonly used in Agile software development to estimate the level of effort needed to implement a feature or change.
+
+Red Hat Application Migration Toolkit uses story points to express the level of effort needed to migrate particular application constructs, and the application as a whole. It does not necessarily translate to man-hours, but the value should be consistent across tasks. 
+
+## How Story Points are Estimated in Rules
+Estimating the level of effort for the story points for a rule can be tricky. The following are the general guidelines RHAMT uses when estimating the level of effort required for a rule.
+
+| Level of Effort | Story Points  | Description                                                                                         | 
```

Comment:
Check the formatting of your table here. It is not getting rendered as a table by Hugo.